packages/coinjoin/src/constants.ts

```diff
@@ -19,6 +19,12 @@ export const STATUS_TIMEOUT = {
 // we are expecting phase to change but server didn't propagate it yet
 export const ROUND_REGISTRATION_END_OFFSET = 2000;
 
+// following https://github.com/zkSNACKs/WalletWasabi/blob/913b16953a965eb922e0ae02a0f71a991a7b9bb7/WalletWasabi/WabiSabi/Client/CoinJoinClient.cs#L31-L34
+// Maximum delay when spreading the requests in time, except input registration requests which
+// timings only depends on the input-reg timeout and signing requests which timings must be larger.
+// This is a maximum cap the delay can be smaller if the remaining time is less.
```

Comment:
It is implemented slightly different from wabisabi so this comment is a bit confusing. The difference is that they always add some random delay even if remaining time is less than this constant https://github.com/zkSNACKs/WalletWasabi/blob/913b16953a965eb922e0ae02a0f71a991a7b9bb7/WalletWasabi/WabiSabi/Client/CoinJoinClient.cs#L669 but here we set 0 delay https://github.com/trezor/trezor-suite/pull/8834/files#diff-f3e642fd48cbec88e803916118d6c2d3a94035e1aa18cf6a1c87073559b7035cR124

But I believe it's ok to keep it as it is implemented now.

Reply:
i believe that if you have 10 or less seconds to send the request(s) playing with the delay is a high risk (banned input and broken round for everyone)

i think network latency will add some delay(s) anyway.

Comment changes form "can be smaller" to "can be zero"